templates/organization/planning/_availabilities_users.html.twig

```diff
@@ -17,7 +17,7 @@
 
 {% block itemDataRowHeader %}
     {{ item.entity }}
-    {{ item.entity.skillSet | filter(skill => skill != type and skill in importantSkillsToDisplay) | map(skill => skill|skillBadge) | join | raw }}
+    {{ item.entity.skillSet | filter(skill => skill != type and skill in skills_to_display) | map(skill => skill|skillBadge) | join | raw }}
```

Comment:
I don't think it's the best way to do it, as we already have the following values:

```yaml
    app.most_important_skills_limit: 6 # used for separing data in the planning
    app.most_important_skills_to_display: 3 # used for displaying a badge near the user name
```

The logic must be done in `SkillSetDomain`:

```php
    public function getSkillsToDisplay(): array
    {
        return \array_slice($this->getSkillSetKeys(), 0, $this->importantSkillsToDisplayLimit);
    }
```

We can maybe replace the `int most_important_skills_to_display` value by an array ?

Caution: are you sure we NEVER want to display other skills ?

Reply:
The other skills will still be available by clicking on "show more informations" so that's not a big issue not to have them at first glance, and we also have the information with the black background cells.

Replacing `most_important_skills_to_display` by an array was my first idea but I wasn't sure  not to break something in don't know about. I'il give this a new try.